ajax/libs/DisplayJS/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "DisplayJS",
+  "filename": "display.min.js",
+  "version": "0.3.2",
+  "description": "A simple JavaScript framework for building ambitious UIs",
+  "keywords": [
+    "Display.JS",
+    "display",
+    "DOM",
+    "manipulation",
+    "framework",
+    "javascript"
+  ],
+  "author": "Arthur Guiot",
+  "license": "MIT",
+  "homepage": "https://display.js.org",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/arguiot/DisplayJS.git"
+  },
+  "npmName": "display.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
Oh 😮, I think 🤔 I actually didn’t put the right build in the `0.3.2` folder, don’t worry 😉, I’ll probably fix it. The `.es6` is a version of DisplayJS that is written in modern JavaScript (ECMAScript 6) which is faster in modern browsers (Chrome, Safari, etc....), but not supported everywhere. So, by default, the file to be served is obviously 🙄 the `.min` fils because it’s the non es6 file that is minified. Unfortunately, I can’t build `.es6.min` files with UglifyJS. **All** the file in the folder is important, so we can’t remove them 😊.

Normally, after I’ll dix the version bug, you’ll be able to merge this PR 😊